dissect/target/tools/yara.py

```diff
@@ -45,8 +45,8 @@ def main():
         parser.exit(1)
 
     try:
-        for target in Target.open_all(args.targets):
-            target.log.info("Scanning target")
+        for target in Target.open_all(args.targets, args.children):
+            target.log.info("Scanning target %s", target)
```

Comment:
Because you use `target.log`, it should already include the target repr in the log line, does logging it again really help?

Reply:
Now you removed the entire log line, was that intended?